drw.c

```diff
@@ -18,6 +18,30 @@ static const unsigned char utfmask[UTF_SIZ + 1] = {0xC0, 0x80, 0xE0, 0xF0, 0xF8}
 static const long utfmin[UTF_SIZ + 1] = {       0,    0,  0x80,  0x800,  0x10000};
 static const long utfmax[UTF_SIZ + 1] = {0x10FFFF, 0x7F, 0x7FF, 0xFFFF, 0x10FFFF};
 
+#if ALPHA_PATCH
+unsigned short
+hextodec(const char *hex)
```

Comment:
You could just use `strtol` with a base of 16.

Reply:
Thanks, I've implemented that.